src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Remove Https configuration
+REM Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+netsh http delete sslcert ipport=0.0.0.0:8084
```

Comment:
No the ports won't be passed via TestProperties from CMD to CMD.  CMD files do not have access to the MsBuild properties unless we pass them explicitly. We can deal with this when we make ports configurable.  I'm just observing not doing this soon is accumulating more debt as we go, so we may want to increase the priority of that discussion.